default.nix

```diff
@@ -1,7 +1,7 @@
 { pkgs ? import <nixpkgs> {}, revCount, shortRev }:
 let
   lib = pkgs.lib;
-  sources = lib.sourceFilesBySuffices ./. [ ".xml" ".txt" ];
+  sources = lib.sourceFilesBySuffices ./. [ ".xml" ".txt" ".bash" ];
```

Comment:
I wanted nice syntax highlighting in my snippets.

Reply:
I'd also like to use `.nix`, but we should still filter out the top-level Nix files.